client/src/reader/components/annotation/popup/menus/Share.js

```diff
@@ -31,10 +25,27 @@ function ShareMenu({
     actions.openCitationDrawer();
   }
 
-  function trackSocialEvent() {
+  function trackShareEvent() {
     trackEvent("share", section.type, section.id);
   }
 
+  const { onClick: onShareClick, canRender: canShare } = useShare(
+    section.attributes.name,
```

Comment:
Can we have this match the `<title>` of the page? `“Introduction” in “The Autobiography of Benjamin Franklin” | Manifold Scholarship` is a much more helpful title than `Introduction`, for instance.

Reply:
Let's do the same on resources.